agent/denylist.go

```diff
@@ -4,24 +4,24 @@ import (
 	"github.com/armon/go-radix"
 )
 
-// Blacklist implements an HTTP endpoint blacklist based on a list of endpoint
+// Denylist implements an HTTP endpoint denylist based on a list of endpoint
 // prefixes which should be blocked.
-type Blacklist struct {
+type Denylist struct {
```

Comment:
🤔 I didn't realise we had actual code types that needed changing. That said, I think this is OK since it's not in our `api` package and we don't guarantee compatibility for core consul module being imported.

If people are importing our main Consul module packages though (which Nomad still does) this potentially breaks binary compatibility. I think that's OK in this case - it's very unlikely this particular object is relied on by anything else and it's not officially supported if it is anyway.

Reply:
Thanks for the review! This was something I was concerned of. I'll drop by the Nomad team and see what they say.